editor/src/clj/editor/editor_extensions/graph.clj

```diff
@@ -0,0 +1,154 @@
+;; Copyright 2020-2024 The Defold Foundation
+;; Copyright 2014-2020 King
+;; Copyright 2009-2014 Ragnar Svensson, Christian Murray
+;; Licensed under the Defold License version 1.0 (the "License"); you may not use
+;; this file except in compliance with the License.
+;;
+;; You may obtain a copy of the License, together with FAQs at
+;; https://www.defold.com/license
+;;
+;; Unless required by applicable law or agreed to in writing, software distributed
+;; under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+;; CONDITIONS OF ANY KIND, either express or implied. See the License for the
+;; specific language governing permissions and limitations under the License.
+
+(ns editor.editor-extensions.graph
+  "Code for interacting with graph from editor scripts, i.e. get/set values"
+  (:require [clojure.spec.alpha :as s]
+            [clojure.string :as string]
+            [dynamo.graph :as g]
+            [editor.code.util :as code.util]
+            [editor.defold-project :as project]
+            [editor.editor-extensions.validation :as validation]
+            [editor.outline :as outline]
+            [editor.properties :as properties]
+            [editor.resource :as resource]
+            [editor.types :as types])
+  (:import [clojure.lang MultiFn]
+           [org.luaj.vm2 LuaError]))
```

Comment:
Moved the code here from `editor_extensions.clj`. No changes in logic here, I only added docstrings.